app/templates/components/modals/theme-warnings.hbs

```diff
@@ -17,7 +17,9 @@
             </div>
         {{/if}}
         {{#each fatalErrors as |error|}}
-            {{gh-theme-error-li error=error}}
+            <div class="theme-validation-item">
```

Comment:
Not too sure about the HTML here, the only direct child that is allowed in a `<ul>` element is an `<li>` element 🤔 If it's working across the browsers we care about then it's probably fine for now but something we should come back to.

Ref: https://html.spec.whatwg.org/multipage/grouping-content.html#the-ul-element